### PR TITLE
Multidocument uploads

### PR DESCRIPTION
**Pull request by cessien**
This branch supports multi document uploads. User presses upload multiple files and is prompted with a tabular screen where you can choose multiple files and specify names and descriptions, as defined in #523 
All Unit tests seemed to pass. I could add more (with a little guidance). All feedback is appreciated.

### CODE REVIEW

**Comment by seven1m** [comment]
@cessien this looks great!!

Would you mind adding a spec for the controller? I think (hope) it will be mostly copy-paste from the existing specs for this controller. If you get stuck, let me know and I'll help out!

I'm nervous if _one_ of the documents in the set fails validation, then the user doesn't know which ones were successful and which ones were not. A spec for that scenario would make me feel better. :smile: 

Again, great work. Looking forward to merging this!

**Comment by cessien** [comment]
Hey @seven1m adding the rspec was easy (rspec is pretty cool!) but you are indeed right with the case of one document failing. I am adding logic to the UI to show the error and the documents that did not upload

File: app/controllers/documents_controller.rb [hunk 1]
```diff
@@ -39,6 +39,30 @@ def create
       else
         render action: 'new_folder'
       end
+    elsif params[:document][:file].kind_of?(Array)
```
Comment (Tr4pSt3R) [inline app/controllers/documents_controller.rb]: Hey @cessien this controller will definitely need to be refactored. I will give a hand when I am free perhaps?

Comment (cessien) [inline app/controllers/documents_controller.rb]: @Tr4pSt3R @seven1m Yeah gladly, I'm still in the process of learning ruby paradigms and I'd gladly take any help

**Comment by cessien** [comment]
@tim I am good to go with this PR! Unless you want me to pull the heavy controller logic in another class (in this pull)! I would need some psuedo examples on how clean you want it to be (ruby n00b)

**Comment by seven1m** [comment]
Awsome I'll get this merged in soon!

**Pull request merged**